src/modules/UI/components/ControlPanel/ControlPanel.ui.js

```diff
@@ -25,7 +27,8 @@ export type Props = {
   username: string,
   openSelectUser: () => void,
   closeSelectUser: () => void,
-  usersView: boolean
+  usersView: boolean,
+  exchangeRate: Object | number
```

Comment:
It seems to always be a number? Maybe it should be `number | null`?

Reply:
The `getExchangeRate` returns `0` if there is no exchange rate, so we'll just use `number`